File: app/src/main/java/com/example/cobe/cryptonews/model/ArticlesResponse.java
package com.example.cobe.cryptonews.model;

import java.util.List;

/**
 * Created by cobe on 29/03/2018.
 */

public class ArticlesResponse {

    private String status;
    private int totalResults;
    private List<Article> articles;

    public List<Article> getArticles() {
        return articles;
    }

}

File: app/src/main/java/com/example/cobe/cryptonews/model/ArticlesResponse.kt
package com.example.cobe.cryptonews.model

import com.example.cobe.cryptonews.common.extensions.Mappable

/**
 * Created by cobe on 13/04/2018.
 */

data class ArticlesResponse(var articles: List<Article>) : Mappable<List<Article>> {

    override fun mapToData(): List<Article> = articles
}
